lasagne/layers/conv.py

```diff
@@ -80,6 +82,74 @@ def conv_output_length(input_length, filter_size, stride, pad=0):
     return output_length
 
 
+def conv_input_length(output_length, filter_size, stride, pad=0):
+    """Helper function to compute the input size of a convolution operation
```

Comment:
To me 'output size of a transposed convolution operation' would be more clear, but it's very subjective.